iphone/Classes/TiMediaVideoPlayerProxy.m

```diff
@@ -883,10 +801,6 @@ - (void)handleLoadStateChangeNotification:(NSNotification *)note
         [self fireEvent:@"load" withObject:nil];
       }
 
-      if ([self _hasListeners:@"preload"]) {
-        DEPRECATED_REPLACED(@"Media.VideoPlayer.preload", @"7.0.0", @"Media.VideoPlayer.load");
```

Comment:
our apidocs do not note this as deprecated, and it seems to be supported on Android? (though looking at the impl, they just fire at the same time_.